.github/workflows/flake-update.yml

```diff
@@ -0,0 +1,28 @@
+name: "Update flakes"
+
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    # 01:15 every monday
+    - cron:  '15 1 * * 1'
+
+jobs:
+  lockfile:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Install Nix
+        uses: cachix/install-nix-action@v16
+        with:
+          extra_nix_config: |
+            access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
+      - name: Update flake.lock
+        uses: DeterminateSystems/update-flake-lock@v14
+        with:
+          pr-title: "Update flake.lock" # Title of PR to be created
```

Comment:
Can you only update nixpkgs here?

Example: https://github.com/DeterminateSystems/update-flake-lock#example-updating-specific-inputs